evm_loader/lib/src/types/tracer_ch_db.rs

```diff
@@ -71,14 +72,41 @@ impl SlotParent {
     }
 }
 
-#[derive(Debug, Row, serde::Deserialize, Clone)]
+#[derive(Row, serde::Deserialize, Clone)]
 pub struct AccountRow {
-    owner: Vec<u8>,
-    lamports: u64,
-    executable: bool,
-    rent_epoch: u64,
-    data: Vec<u8>,
-    txn_signature: Vec<Option<u8>>,
+    pub owner: Vec<u8>,
+    pub lamports: u64,
+    pub executable: bool,
+    pub rent_epoch: u64,
+    pub data: Vec<u8>,
+    pub txn_signature: Vec<Option<u8>>,
+}
+
+impl fmt::Display for AccountRow {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(
+            f,
+            "AccountRow {{\n    owner: {},\n    lamports: {},\n    executable: {},\n    rent_epoch: {},\n}}",
+            bs58::encode(&self.owner).into_string(),
+            self.lamports,
+            self.executable,
+            self.rent_epoch,
+        )
+    }
+}
+
+impl<'a> fmt::Debug for AccountRow {
```

Comment:
Nitpick: I think this lifetime can be removed.

Reply:
Done.